docker/README.md

```diff
@@ -0,0 +1,148 @@
+# Docker and exercism.io
+
+Optionally, one can develop (and even deploy) exercism.io using a tool
+called Docker. The primary advantage of using it is that you don't have to
+install a ton of dependencies--like a ruby version manager, ruby, postgres,
+nodejs, and a bunch of Ruby gems--on your own system. The disadvantage,
+however, is that you may need to acquaint yourself with Docker, and you
+need a 64-bit machine.
+
+For a basic overview, see [What is Docker?][] on the Docker website.
```

Comment:
Oh!  Yeah, since READMEs are potentially read by devs in their text editors rather than rendered as Markdown, I try to make their source as human-readable as possible, so I like leaving URLs as footnotes so they don't interrupt the flow of the text. But I can switch to inline URLs if you prefer those, too--whatever works!